alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -495,6 +497,125 @@ fn box_drawing(character: char, metrics: &Metrics, offset: &Delta<i8>) -> Raster
     }
 }
 
+fn powerline_drawing(character: char, metrics: &Metrics, offset: &Delta<i8>) -> RasterizedGlyph {
+    let height = (metrics.line_height as i32 + offset.y as i32) as usize;
+    let width = (metrics.average_advance as i32 + offset.x as i32) as usize;
+    // Use one eight of the cell width, since this is used as a step size for block elements.
+    let stroke_size = cmp::max((width as f32 / 8.).round() as usize, 1) as f32;
+
+    let mut canvas = Canvas::new(width, height);
+
+    let x_end = width as f32 - 1.;
+    let y_end = height as f32 - 1.;
+    let y_center = y_end / 2.;
+
+    // Offset to achieve 90 degree angle in the tip when we can do so just by moving 1px down/up
+    // on the opposite side. When we can't do so, just cut on the tip side.
+    let y_offset = if (height + 1).saturating_sub(2 * width) / 2 == 1 { 1 } else { 0 };
```

Comment:
This is optional, it might be simpler and easier to maintain if we just always do this.